opentera-webrtc-native-client/OpenteraWebrtcNativeClient/include/OpenteraWebrtcNativeClient/StreamClient.h

```diff
@@ -55,6 +55,7 @@ namespace opentera
         void muteLocalAudio();
         void unmuteLocalAudio();
         void setLocalAudioMuted(bool muted);
```

Comment:
Should match the local interface and add `isRemoteAudioMuted`, `muteRemoteAudio` and `unmuteRemoteAudio`?

Reply:
`isRemoteAudioMuted`, `muteRemoteAudio` and `unmuteRemoteAudio` wouldn't be used, should I still make them?

Reply:
I think that you should. @doumdi, what do you think?

Reply:
I think we should match the interface because we are creating a general library and I think getters can be useful on the future.

Reply:
I've added them